Mathlib/Data/DFinsupp/Order.lean

```diff
@@ -64,12 +63,12 @@ theorem orderEmbeddingToFun_apply {f : Π₀ i, α i} {i : ι} :
 end LE
 
 section Preorder
+
 variable [∀ i, Preorder (α i)] {f g : Π₀ i, α i}
 
-instance : Preorder (Π₀ i, α i) :=
-  { (inferInstance : LE (DFinsupp α)) with
-    le_refl := fun f i ↦ le_rfl
-    le_trans := fun f g h hfg hgh i ↦ (hfg i).trans (hgh i) }
+instance : Preorder (Π₀ i, α i) where
+  toLE := inferInstance
```

Comment:
I believe this works
```suggestion
```

Reply:
I wanted to be sure that the next `__` won't override it with something that is defeq but isn't reducibly defeq.